tests/helpers/test_json.py

```diff
@@ -335,3 +341,53 @@ def as_dict(self) -> dict[str, Any]:
         BadData(),
         dump=partial(json.dumps, cls=MockJSONEncoder),
     ) == {"$(BadData).bla": bad_data}
+
+
+def test_deprecated_json_loads(caplog: pytest.LogCaptureFixture) -> None:
+    """Test deprecated json_loads function.
+
+    It was moved from helpers to util in #88099
+    """
+    json_helper.json_loads("{}")
+    assert (
+        "json_loads is a deprecated function which will be removed in "
+        "HA Core 2025.2. Use homeassistant.util.json.json_loads instead"
+    ) in caplog.text
+
+
+@pytest.mark.parametrize(
+    ("constant_name", "replacement_name", "replacement", "breaks_in_ha_version"),
```

Comment:
```suggestion
    ("constant_name", "replacement_name", "replacement"),
```

`breaks_in_ha_version` is the same for all tests, so we don't need to parameterized it